LayoutTests/svg/filters/feMerge-wrong-input.svg

```diff
@@ -1,7 +1,7 @@
 <svg xmlns="http://www.w3.org/2000/svg">
 <defs>
 <!-- check that WebKit doesn't crash, if an effect is not available in feMerge -->
-<filter id="filter" primitiveUnits="objectBoundingBox">
+<filter id="filter" filterUnits="userSpaceOnUse" primitiveUnits="objectBoundingBox">
```

Comment:
Any particular reason why you have to modify this?

Reply:
So original Blink's patch added `viewport.isEmpty()` (which I didn't added in first place), because I knew it would cause to another issue, which they fixed in following PR:

https://source.chromium.org/chromium/chromium/src/+/8ff0050993b2d2654c9f3170e789e65b0b303691

So in the above fix, they have to adjust this test as well. So that's what I did.